### PR TITLE
Added a Paranormal Investigator Profession

### PR DESCRIPTION
**Pull request by emarinier**
#### Summary
Content "Added a Paranormal Investigator profession"

#### Purpose of change

I wanted to role-play as a paranormal investigator, but found the closest profession to be photojournalist, which didn't quite capture the Ghostbusters-style profession that I wanted. A paranormal investigator seems appropriate for the setting, given the zombies, aliens, and other supernatural creatures.

#### Describe the solution

Adds a Paranormal Investigator profession, largely inspired by the characters in the movie Ghostbusters: non-religious paranormal investigators that get messy and have lots of gadgets.

#### Describe alternatives you've considered

Using the existing Photojournalist (which focuses more on journalism) or Hazmat Unit (which focuses more on medicine) professions.

I tried to select starting items that would feel like "gadgets", without going into bionics or making the start too powerful. I selected thermometer and radiation biomoniter, but I'm open to alternatives.

I chose the profession cost (2) to be similar to other professions, namely somewhere between Photojournalist (0) and Lab Technician (3).

#### Testing

I have created multiple characters using the new profession and it appears to be working as intended.

#### Additional context

paranormal1

paranormal2

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -4057,5 +4057,40 @@
       "female": [ "bra", "panties" ]
     },
     "traits": [ "LIAR" ]
+  },
+  {
+    "type": "profession",
+    "id": "paranormal_investigator",
+    "name": "Paranormal Investigator",
+    "description": "You were dismissed from your position as a parapsychology professor at the local university and started your own business as a paranormal invesigator. Your former colleagues didn't approve of your research, but it looks like it's paying off now.",
```
Comment (BrettDong) [inline data/json/professions.json:4065]:
```suggestion
    "description": "You were dismissed from your position as a parapsychology professor at the local university and started your own business as a paranormal invesigator.  Your former colleagues didn't approve of your research, but it looks like it's paying off now.",
```

**Comment by Entity-y** [comment]
I think a paranormal investigator would carry a radio of some kind to try to communicate with ghosts. They would certainly carry a camcorder (which doesn't exist in game), a prosnap camera might be a decent alternative.

**Comment by emarinier** [comment]
> 
> 
> I think a paranormal investigator would carry a radio of some kind to try to communicate with ghosts. They would certainly carry a camcorder (which doesn't exist in game), a prosnap camera might be a decent alternative.
> 
> It would also make sense for the character to have some degree of schizophrenia to be chasing ghosts.

I agree with adding the radio and camera. I've removed the thermometer and added those two tools.

I'm a bit reluctant to add something like schizophrenia, because I don't like the idea that the profession is necessarily tried to it. I agree that certain kinds of people would be drawn to the profession, but I don't know if schizophrenia captures it.

**Comment by FoolsGold45** [comment]
Spiritual sounds like it would be very fitting as a profession trait.

Might I suggest swapping the cell phone with a smartphone? #51734 updates most of the existing professions to start with one instead of a cell phone, it'd be good to keep up that precedent unless your profession has a good reason to prefer cell phones (maybe 5G waves make the ghosts stronger?!)

**Comment by emarinier** [comment]
> Spiritual sounds like it would be very fitting as a profession trait.
> 
> Might I suggest swapping the cell phone with a smartphone? #51734 updates most of the existing professions to start with one instead of a cell phone, it'd be good to keep up that precedent unless your profession has a good reason to prefer cell phones (maybe 5G waves make the ghosts stronger?!)

Absolutely. I've added the spiritual trait and swapped the cellphone for a smartphone.

**Review by kevingranade — APPROVED** [review]
LGTM, nicely thematic without being too 4th wall breaking or OP.
(only holdup is the tests)

**Comment by emarinier** [comment]
Thank you for finding the source of the error. I'm still new to contributing, so I asked about the error in dev-help on the Discord a little while ago, and wist said the error was unrelated to my changes.

Should I create an issue that reports this error? Let me know if there's anything I should do to help resolve the error! Thanks. :)

**Pull request merged**